pkgs/applications/networking/mailreaders/himalaya/default.nix

```diff
@@ -10,37 +10,45 @@
 , libiconv
 , openssl
 , notmuch
+, withRusttlsTls ? true
+, withRusttlsNativeCerts ? withRusttlsTls
+, withNativeTls ? false
+, withNativeTlsVendored ? withNativeTls
 , withImapBackend ? true
 , withNotmuchBackend ? false
 , withSmtpSender ? true
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "himalaya";
-  version = "0.7.1";
+  version = "0.7.3";
 
   src = fetchFromGitHub {
     owner = "soywod";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-yAfNH9LSXlS/Hzi5kAuur5BX2vITMucprDzxhlV8RiY=";
+    sha256 = "sha256-HmH4qL70ii8rS8OeUnUxsy9/wMx+f2SBd1AyRqlfKfc=";
```

Comment:
Minor, _minor_ nit: this could be `"HmH4qL70ii8rS8OeUnUxsy9/wMx+f2SBd1AyRqlfKfc=";` the leading `sha256-` is redundant.

Reply:
Oh great, I did not know about this. Thank you!